pysrc/bytewax/connectors/bigquery/__init__.py

```diff
@@ -0,0 +1 @@
+from .bigquery_output import BigqueryOutputConfig  # noqa: F401
```

Comment:
What's the thinking behind calling the submodule `bigquery_output`? The full path would then be `connectors.bigquery.bigquery_output`. I think we could drop the second `bigquery` for just `connectors.bigquery.output`.